src/interfaces/WF/WFIPAddressType.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * @typedef {('IPv4'|'IPv6')} WFIPAddressTypeOption
```

Comment:
Could you remove this whole typedef comment please? This is my fault - I meant to remove all of these when I was changing how the docs are generated a few days ago, clearly I missed a couple! They're no longer needed now that we've switched to TypeDoc to generate the docs.

I'm going to remove all of the remaining `@typedef` comments I left in by mistake, so if you could just remove them from this PR then that'll save me a job 🙂